lib/cl/server.lisp

```diff
@@ -0,0 +1,230 @@
+(in-package #:org.apache.thrift.implementation)
```

Comment:
Does one need the ".implementation" in the namespace?

Reply:
it is a common practice in Common Lisp to have a package dedicated for usage by a programmer (which exports the protocol symbols - library API that is) and having no other symbols of its own and an implementation package which may have other internal symbols and interfaces. This allows avoiding for instance symbol name conflicts (for instance list vs list).